core/tests/webdriverio_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -151,15 +152,18 @@ var ExplorationEditorFeedbackTab = function() {
     await action.setValue(
       'Feedback Response Text Area',
       feedbackResponseTextArea, feedbackResponse);
-    await action.click('Feedback Status Dropdow', feedbackStatusDropdown);
-    var optionLabelFeedbackStatus = (
-      $(`option[label="${feedbackStatus}"]`));
-    await action.click(
-      'Option[label = "feedback status"', optionLabelFeedbackStatus);
+    await waitFor.presenceOf(
+      feedbackStatusDropdown, 'Feedback Status Dropdown is not visible');
+    await feedbackStatusDropdown.selectByVisibleText(feedbackStatus);
     await action.click(
       'Feedback Send Response Button', feedbackSendResponseButton);
   };
 
+  this.expectNumberOfFeedbackMessagesToBe = async function(number) {
+    await waitFor.numberOfElementsToBe(
+      feedbackMessageLocator, 'Feedback message', number);
```

Comment:
We are using class-name because in `numberOfElementsToBe` we need classname in order to use `$$` again and again till the number of elements are equal to the given number.